dev/set_permissions.sh

```diff
@@ -21,74 +21,75 @@
 
 # Default is alignak account
 ACCOUNT=$1
+# Default is /usr/local account
+PREFIX=$2
 # Parse command line arguments
 if [ $# -eq 0 ]; then
     ACCOUNT="alignak"
+    PREFIX="/usr/local"
```

Comment:
You will have problem if you pass only first argument and not the second

Reply:
Sure! I know ... I will add one more test in the script!